subsys/net/l2/openthread/Kconfig

```diff
@@ -187,12 +187,15 @@ menuconfig OPENTHREAD_NCP
 	help
 	  Enable NCP in OpenThread stack.
 
-if OPENTHREAD_NCP
-
-config OPENTHREAD_NCP_RADIO
-	bool "Network Co-Processor as Radio"
+menuconfig OPENTHREAD_RCP
+	bool "Radio Co-Processor"
+	select OPENTHREAD_MANUAL_START
```

Comment:
I'd add `depends on !OPENTHREAD_NCP`, since the options are exclusive. And vice versa in the `OPENTHREAD_NCP` options, `depends on !OPENTHREAD_RCP`. This way, if one option is selected, the other one will be disabled.

Reply:
Will go for the `choice` OPENTHREAD_COPROCESSOR_NCP / OPENTHREAD_COPROCESSOR_RCP.